xwiki-platform-core/xwiki-platform-ckeditor/xwiki-platform-ckeditor-plugins/src/main/webjar/xwiki-icon/iconService.js

```diff
@@ -52,6 +52,7 @@ define('xwiki-iconService', [
       iconsPromise = iconsPromisesPerTheme[query] = $.getJSON(getResourceURL('data_icons', {
         iconTheme,
         query,
+        'metadata': 'true'
```

Comment:
OK, let's keep it like this then, and add a note in the RN.

Reply:
I think we should cache the icon metadata, and make it fast through the cache, not by excluding the metadata by default.

Reply:
For me the priority should be to add some kind of pagination to the icon picker because it doesn't scale currently. Adding the icon metadata cache is nice, but I'm not sure if it will solve the slowness (for an image-based icon theme like Silk where the icon picker needs to fetch and display 1k+ image icons).